src/components/content/conditions/patient-conditions-all.tsx

```diff
@@ -51,7 +53,11 @@ function PatientConditionsAllComponent({
   const hasZeroFilteredRecords = !isEmptyQuery && data.length === 0;
 
   const empty = (
-    <EmptyPatientTable hasZeroFilteredRecords={hasZeroFilteredRecords} resourceName="conditions" />
+    <EmptyPatientTable
+      hasZeroFilteredRecords={hasZeroFilteredRecords}
+      resourceName="conditions"
+      trackInteraction={trackInteraction}
+    />
```

Comment:
This can be omitted if EmptyPatientTable uses the `useAnalytics` hook itself.
```suggestion
    <EmptyPatientTable
      hasZeroFilteredRecords={hasZeroFilteredRecords}
      resourceName="conditions"
    />
```